mars/dataframe/utils.py

```diff
@@ -1424,3 +1431,44 @@ def _concat_chunks(merge_chunks: List[ChunkType], output_index: int):
     else:
         params["nsplits"] = (tuple(n_split), df_or_series.nsplits[1])
     return new_op.new_tileable(df_or_series.op.inputs, kws=[params])
+
+
+def clean_up_func(op):
+    closure_clean_up_bytes_threshold = int(
+        os.getenv("MARS_CLOSURE_CLEAN_UP_BYTES_THRESHOLD", 10**4)
+    )
+    if closure_clean_up_bytes_threshold == -1:  # pragma: no cover
+        return
+    # note: Vineyard internally uses `pickle` which fails to pickle
+    # cell objects and corresponding functions.
+    if vineyard is not None:
```

Comment:
If `vineyard` is installed but other storage are used, this will skip cleanup too?